libvmaf/src/output.c

```diff
@@ -44,6 +44,26 @@ static const char *pool_method_name[] = {
     [VMAF_POOL_METHOD_HARMONIC_MEAN] = "harmonic_mean",
 };
 
+static int count_leading_zeros_flat(double x)
```

Comment:
nit: I think this used to be `float` and got typoed to `flat`? in either case, it is a bit weird to call it `float` and have it operate on doubles. There are some uses of the "`_s` for single, `_d` for double" convention in this repo, such as [here](https://github.com/Netflix/vmaf/blob/262db8dbdb6210d2f5aaec36b77a1fb76e9c9823/libvmaf/src/feature/adm_tools.h#L278-L280) - maybe `count_leading_zeros_d`?

Reply:
welp, that was definitely a typo. Renamed to `count_leading_zeros_d` as suggested.